src/org/labkey/test/params/FieldDefinition.java

```diff
@@ -313,13 +314,19 @@ public FieldDefinition setPrincipalConceptCode(String principalConceptCode)
         return this;
     }
 
+    public String getPrincipalConceptSearchSourceOntology()
+    {
+        return _principalConceptSearchSourceOntology;
+    }
+
     public String getPrincipalConceptSearchExpression()
     {
         return _principalConceptSearchExpression;
     }
 
```

Comment:
I think it will be helpful to note that the searchExpression is to support the UI case/not be passed in as json to the remoteAPI.  We only use the searchExpression to give the UI the ability to obtain the conceptCode, when we're using the API we give it the conceptCode directly